### ISSUE TITLE
Cached tiles for recent changes on OSM.

### ISSUE THREAD

**Issue by Mrjuanblack** [vvoovv/blender-geo#396]
Hello, I was using leaflet and OSM to create a web page with a map for a future urbanization. I had to edit a river directly on OSM because it was going right on top of the blueprints we had, and that river changed it's shape long ago.
I had some issues visualizing the changes I did, but after pressing ctrl+F5 multiple times I no longer saw some cached tiles on the web page. I'm using blender to export some art to overlay on top of the map and I thought of using this addon to stop having issues trying to aling my 3D models with the online map, but when I use the Image Overlay option I get the same error of some tiles being the old ones (like a cache issue).
I'm not sure if I can "ctrl+f5" with this addon, and this issue only happens when the zoom is at level 18. 19 works just fine.
Is it an issue on OSM side? Or can I do something to fix this?
![image](https://github.com/vvoovv/blosm/assets/48628834/2cc1a285-3b93-462b-8c10-06b1d2d26501)

**Comment by vvoovv** [comment]
Hi @Mrjuanblack 

Tiles are cached by the addon. A manual delete will be required for your case.

Find the needed subfolder in the subfolder _overlay_ of the directory set in the addon preferences for the addon and delete that subfolder.

**Comment by Mrjuanblack** [comment]
Thanks for the quick response! I tried that too before writing this question yesterday and it did not work. There's no variation in the tiles when deleting the subfolder and importing the image overlay again.
But compared to yesterday, one tile in the river is fixed now, however, I would not like to wait 13 hours for each tile to be correctly displayed.
This is the imported image now:
![image](https://github.com/vvoovv/blosm/assets/48628834/4a459a54-14e9-4cc6-9945-3aa9057bc5da)
This is how it should be displayed (from OSM web page):
![image](https://github.com/vvoovv/blosm/assets/48628834/ef39ad80-ba4b-4193-8ff7-9b2be43ff964)

**Comment by vvoovv** [comment]
Sorry, we don't have access to tile servers or cache servers.

You could manually replace the tiles in question in the overlay subfolder.

**Comment by Mrjuanblack** [comment]
I'm not sure how I would go about downloading specific tiles, but right now the import image overlay is working without issues.
Since this can't be resolved from this side, I think we can close this, but for other people that might have this issue:
- OSM took a while (1 day) to render properly the changes I did to a place I would guess is not viewed by many (might be the reason why I had issues with cached images for a whole day).
- After that first day, downloading the tiles like I did using this tool still gave me similar issues. I would say that after 2-3 days since you made the changes in the map, the tiles will be downloaded correctly (again, it might vary depending on the location of the changes from what I have read online).

Just remember to delete the subfolder as @vvoovv said to check if downloading the tiles again will work!

Thanks for this amazing tool :)
